src/main/resources/application-development.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  config:
+    activate:
+      on-profile: development
+
+application:
+  settings:
+    serverPort: 8088
+    openWeatherMap:
+      api:
+        key: 675bf40b4d52b5fb5e47199ec8315931
+        weatherUrl: https://api.openweathermap.org/data/2.5/weather
+        forecastUrl: https://api.openweathermap.org/data/2.5/forecast
+    yandexMap:
+      api:
+        key: 675bf40b4d52b5fb5e47199ec8315931
```

Comment:
key for the yandex is the same as the key for open-weather. If you currently don't have a value for this property you can name it like ''